features/index/index_as_table.feature

```diff
@@ -289,3 +289,24 @@ Feature: Index as Table
       | Id | Title        | Title Length |
       | 1 | Hello World   | 11 |
       | 2 | Bye bye world | 13 |
+
+  Scenario: Show footer row
+    Given a post with the title "Hello World" exists
+    And a post with the title "Bye bye world" exists
+    And an index configuration of:
+      """
+        ActiveAdmin.register Post do
+
+          index has_footer: true do
+            column :id, footer: 'Total:'
+            column :category, footer: ->(collection) { collection.pluck(:id).sum }
+            column :title, footer: :count
```

Comment:
@ukolovda  can you please add test with example where we ave more than 1 page to demonstrate how it works in real life, as far as i remember collection here has already applied limit and offset parts, so it can calculate count on paginated collection not on filtered collection